### ISSUE TITLE
IV% innacurate

### ISSUE THREAD

**Issue by shiznid12** [PocketMobsters/PokeMobBot#384]
According to multiple programs, as well as IV finding websites, the IV% that the bot gives us is quite inaccurate.

I am not sure if the bot is wrong, or these programs are wrong.. but I have a "97% Chansey" that is actually 78%. PokeAssistant.com also states this IV%, as well as Blossoms Pokemon Manager.

Has anyone else acknowledged these discrepancies?

**Comment by Xe1** [comment]
Yes, I noticed that in the latest update. PokeAdvisor.com gives me different IV values than the bot.

**Comment by Emuuung** [comment]
Not only that, the bot gives really weird differences between IV percentage, a pokemon with 44 of 45 total IVs should have 97.77% (rounded to 97.78%), but apparently I have pokemon with 98.37% and 98.32%.... Where is that small difference coming from? How is the IV percentage being calculated?

**Comment by eslindsey** [comment]
http://i.imgur.com/FCAd5Xq.png

**Comment by aNNdii** [comment]
+1

**Comment by Tytn** [comment]
I've noticed it as well. Usually small differences, but every once in a while a big one.
